src/sacn/mem.c

```diff
@@ -1082,3 +1490,45 @@ void deinit_source_limit_exceeded_buf(void)
 }
 
 #endif  // SACN_DYNAMIC_MEM
+
+etcpal_error_t init_sources(void)
+{
+  etcpal_error_t res = kEtcPalErrOk;
+
+#if SACN_SOURCE_ENABLED
+#if !SACN_DYNAMIC_MEM
+  if (res == kEtcPalErrOk)
```

Comment:
Since the mempools don't have a deinit function, I like to take advantage of the fact that kEtcPalErrOk is 0 to shorten the code:
```
res |= etcpal_mempool_init(foo);
res |= etcpal_mempool_init(bar);
res |= etcpal_mempool_init(baz);
// ...
```

Reply:
Mempools removed in https://github.com/ETCLabs/sACN/pull/32/commits/9a88cc62e4c3fcb6011291027e0f8274c5237b9a.